lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoAnnotationDeclarationCore.kt

```diff
@@ -39,9 +39,7 @@ internal class KoAnnotationDeclarationCore private constructor(
             ?.name ?: name
     }
 
-    override fun toString(): String {
-        return locationWithText
-    }
+    override fun toString(): String = name
```

Comment:
Do we have to duplicate this logic and override this in so many places? Isn't the `KoBaseProvider` class enough for a single override?